### ISSUE TITLE
A bug in the v1.8.0 for narrative in-text citation

### ISSUE THREAD

**Issue by byzheng** [kookma/TW-Refnotes#25]
In the tiddler

```
See <<ref "bustos-korts_modelling_2016" n>>
See <<ref bustos-korts_modelling_2016 n>>
```

Then in the reference list, I get this error

```
Reference bustos-korts_modelling_2016 n NOT FOUND. Check your input.
Reference bustos-korts_modelling_2016" n NOT FOUND. Check your input.
```

Narrative is only working for  single quotation

```

See <<ref 'bustos-korts_modelling_2016' n>>
```

**Comment by kookma** [comment]
This is by design! See note 3 in https://kookma.github.io/TW-Refnotes/#demo%2Fref-macro

**Comment by kookma** [comment]
Hi @maxsall

**Comment by kookma** [comment]
Mark,
 See example in https://kookma.github.io/TW-Refnotes/#demo%2Fref-macro
  
You  also create a new tiddler with below text:

```
<<ref "rostami-2022" n 'p. 16'>>


<<showrefs>>
```

This does not work, but, the below form will work.

```
<<ref 'rostami-2022' n 'p. 16'>>


<<showrefs>>
```

**Comment by kookma** [comment]
This last one works.
Despite the complexity of code,  I tested all combinations of double quotes / single quotes and double bracket / quotes and all works.

```
<<ref [[rostami-2022]] n 'p. 16'>>
<<ref [[rostami-2022]] n "p. 16">>

<<ref 'rostami-2022' 'n' 'p. 16'>>
<<ref 'rostami-2022' "n" 'pp. 123-345'>>

<<ref "rostami-2022" "n" "pp. 123-345">>
<<ref "rostami-2022" n "pp. 123-345">>
<<ref "rostami-2022" n 'pp. 123-345'>>

<<showrefs>>
```

I use this code in Refnotes 1.8.1, and will push a new update shortly.

**Comment by kookma** [comment]
Fixed in 1.8.1
